core/services/scan.go

```
@@ -300,6 +300,8 @@ func (s *ScanService) ScanCP(mainCtx context.Context) error {
 			}
 		}
 
+		filteredCve := domain.CVEManifest{}
+		cveExceptionsComplete := true
```

golangci-lint (2.12.2)

[error] ineffectual assignment to cveExceptionsComplete

(ineffassign)